src/main/java/macrobase/analysis/classify/DumpClassifier.java

```diff
@@ -35,7 +35,8 @@ public DumpClassifier(MacroBaseConf conf, OutlierClassifier input) throws IOExce
     public DumpClassifier(MacroBaseConf conf, OutlierClassifier input, String name) throws IOException {
         this.conf = conf;
         this.input = input;
-        filepath = String.format("test_out/dump_classifier_%s.txt", name);
+	// TODO: output directory should be configurable
```

Comment:
I dump my files in `target` directory. It's in gitignore and maven seems to put a lot of temporary things there. We could dump all our diagnostic files there too.